patchify

```diff
@@ -264,6 +266,31 @@ def load_patch_definition(filename):
                 definition=definition,
             )
 
+    for key in definition.keys():
```

Comment:
no reason for `.keys()`, just makes for extra allocations

Reply:
It is required, because errors are added to the definition, so without .keys() it causes an error when the dict is modified while being iterated over.